app/mockServices/mockData/services.ts

```diff
@@ -15,6 +15,7 @@ export const servicesData = {
     plans: [
       {
         name: 'rh-ded-topic', osbGuid: '1', displayName: 'Red Hat - Dedicated - Topic', description: '$.65 / 1 Million messages', bullets: ['One', 'Two', 'Three'],
+        bindable: true,
```

Comment:
@dtaylor113 `bindable` is required on the service class. The value on the plan is only needed to override the service class value.

We should add it to the all the mock data service classes. (Does not need to be this PR.)